Sources/BSON/BSONBinary.swift

```diff
@@ -188,4 +188,12 @@ extension BSONBinary: BSONValue {
         }
         buffer.writeBytes(self.data.readableBytesView)
     }
+
+    public init(from decoder: Decoder) throws {
+        throw getDecodingError(type: Self.self, decoder: decoder)
+    }
+
+    public func encode(to: Encoder) throws {
```

Comment:
nit: can we name the encoder rather than just using the `to` label? e.g. `to encoder: Encoder`.  same elsewhere

also a question, since these conformances are all identical can we save ourselves duplication and put the erroring behavior as the default conformance on `BSONValue`?

Reply:
Done and done. 

I had assumed the extension wasn't possible because we didn't do it already, but it seems to work just fine.